backend/graphql/types/testType.ts

```diff
@@ -16,43 +44,61 @@ const testType = gql`
     answerIndex: Float!
   }
 
-  input NumericQuestionMetadataInput {
+  input MultiSelectMetadataInput {
+    options: [String!]!
+    answerIndices: [Float!]!
+  }
+
+  type MultiSelectMetadata {
+    options: [String!]!
+    answerIndices: [Float!]!
+  }
+
+  input ShortAnswerMetadataInput {
     answer: Float!
   }
 
-  type NumericQuestionMetadata {
+  type ShortAnswerMetadata {
     answer: Float!
```

Comment:
Does the linked [slack thread](https://uwblueprint.slack.com/archives/C04E9L71F47/p1675095359429609) imply that this should be an `Int` instead of a `Float`?

Reply:
hmm i'm not sure i understand - we want to accept either integers or decimals as answers

in typescript, we use `Number` type to represent this
in graphql, the equivalent type is `Float`

if the intention is to not accept decimals, we can use `Int`

Reply:
I guess my question is, do we intend to not accept decimals?

Reply:
o, i can confirm with mehak lol

Reply:
mehak confirmed that we want to accept decimals!

Reply:
okay sounds good! don't mind me then :$